### PR TITLE
Git status report, doc update, 1bash healthcheck, firstboot rework

### PR DESCRIPTION
**Pull request by LuKePicci**
This patchset is intended to prepare nvOC to be bundled into pre-built full os images.
Firstboot operations will be moved in proper scripts outside the scope of this repo. Those operations include:
- partition resizie
- nvOC code & miners fetch
- 1bash conversion & import

Note: 1bash healthcheck will be skipped if you have not installed 'shellcheck', should be part of future nvOC images

### CODE REVIEW

**Review by papampi — COMMENTED** [review]
Does it need a firstBOOT script to complete this PR?

**Comment by LuKePicci** [comment]
Nope, but I wouldn't merge until the firstboot stuff is ready on the other repo

Da: Payam Nab
Inviato: martedì 19 giugno 2018 17:57
A: papampi/nvOC_by_fullzero_Community_Release
Cc: Luca Piccirillo; Author
Oggetto: Re: [papampi/nvOC_by_fullzero_Community_Release] Cleanup, doc update,1bash healthcheck, firstboot rework (#99)

@papampi commented on this pull request.
Does it need a firstBOOT script to complete this PR?

**Comment by LuKePicci** [comment]
Ok, this is ready, just waiting for your review and approval. All links and references to 'release' branch will work as soon as a branch with that name exists. Will do it from the current 2.1 as it is now just to not leave broken refs around. Will merge again 2.1 into release when it's released for true.

**Review by papampi — APPROVED** [review]

**Pull request merged**
